packages/jest-matcher-utils/src/deepCyclicCopyReplaceable.ts

```diff
@@ -42,6 +44,9 @@ export default function deepCyclicCopyReplaceable<T>(
     return deepCyclicCopyMap(value, cycles);
   } else if (isBuiltInObject(value)) {
     return value;
+  } else if (plugins.DOMElement.test(value)) {
```

Comment:
```suggestion
  } else if (typeof value.cloneNode === 'function') {
```


maybe? not sure if it's better or not